src/libraries/System.Configuration.ConfigurationManager/src/CompatibilitySuppressions.xml

```diff
@@ -1,5 +1,29 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
```

Comment:
```suggestion
  <!-- Because things in the Internal namespace is considered implementation details, these can safely break. If someone outside of the runtime repository use the types in said namespace they acknowledge that the implementation may change at any time and possibly contain breaking changes. -->
  <Suppression>
```